.github/workflows/push-container.yaml

```diff
@@ -16,20 +16,22 @@ on:
 jobs:
     push-container:
         runs-on: ubuntu-latest
+        env:
+            CAN_PUSH: ${{ (github.event_name == 'push' && github.repository == 'nangohq/nango') || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false) }}
```

Comment:
the logic is a bit more complicated, ie: can push if push from the repo (will be false on fork since github.repository will be different) or if PR and head repo is not a fork